nmmo/render/packet_manager.py

```diff
@@ -0,0 +1,99 @@
+import json
+import lzma
+import logging
+
+from .render_utils import np_encoder, patch_packet
+
+
+class PacketManager:
+  @staticmethod
+  def create(realm):
+    if realm.config.SAVE_REPLAY:
+      return SimplePacketManager(realm)
+
+    return DummyPacketManager()
+
+
+class DummyPacketManager(PacketManager):
+  def reset(self):
+    pass
+
+  def update(self, packet):
+    pass
+
+  def save(self, save_file):
+    pass
+
+
+class SimplePacketManager(PacketManager):
```

Comment:
Should we call this FilePacketManager? or MemoryPacketManager? Simple doesn't tell us much

Reply:
I changed to ReplayFileHelper. FileReplayHelper sounded somewhat odd.